bin/apply.bash

```diff
@@ -28,6 +28,12 @@ update_ips_dryrun() {
   fi
 }
 
+check_migration() {
+  if get_migration_status "${1}" &>/dev/null; then
+    log_fatal "Migration ongoing, try again when it has completed or 'ck8s upgrade unlock'"
+  fi
+}
```

Comment:
Would it make sense to instead say upgrade instead of migration in the cli? It feels more logical when that is the command is upgrade.

Reply:
Makes sense, with upgrade being the overall action and migrations being individual changes applied.

Reply:
```suggestion
check_migration() {
  if get_migration_status "${1}" &>/dev/null; then
    log_fatal "Upgrade ongoing, try again when it has completed or 'ck8s upgrade unlock'"
  fi
}
```